lib/ronin/vulns/web_vuln.rb

```diff
@@ -455,7 +455,12 @@ def self.scan(url, query_params:  nil,
 
           if form_params
             vulns.concat(
-              scan_form_params(url,form_params, http: http, **kwargs,&block)
+              case form_params
+              when true
+                scan_form_params(url,Set.new(kwargs[:form_data]&.keys), http: http, **kwargs,&block)
```

Comment:
Let's move that logic into `self.scan_form_params`, similar to `self.scan_query_params`:

```ruby
def self.scan_form_params(url,form_params=nil, http: nil, form_data: {}, **kwargs)
  ...
  form_params ||= form_data.keys
  ...
end
```

Reply:
Will also need new specs for when `.scan_form_params` is called without any `form_params`, but `form_data:` is given.